packaging/cockpit-podman.spec.in

```diff
@@ -33,6 +33,9 @@ BuildRequires: libappstream-glib-devel
 
 Requires:       cockpit-bridge
 Requires:       podman >= 2.0.4
+%if 0%{?fedora} == 36 || 0%{?fedora} == 37 || 0%{?centos} == 9
```

Comment:
Can you please add a `# HACK:` link to the criu bug report here?

Reply:
Done :)